itests/docker/docker.go

```diff
@@ -175,14 +179,27 @@ func (d *Docker) Finish(cancel context.CancelFunc) {
 	cancel()
 }
 
+func (d *Docker) stopContainer(containerID string) error {
+	const shutdownTimeout = 5 // In seconds.
```

Comment:
I think `shutdownTimeout` is low, it's better to set it, for instance, to 15 seconds.